.cache/v/cache/lastfailed

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
What's this file for?

Reply:
That’s me not noticing  that the gitignorge isn’t ignoring .cache. :/ I’ll fix that.